src/modm/board/disco_f469ni/board.xml

```diff
@@ -4,6 +4,7 @@
       <path>../../../../repo.lb</path>
     </repository>
   </repositories>
+  <!-- <versions>b-03,b-02,b-01</versions> -->
```

Comment:
Uncomment?

Reply:
Somewhere the revisions need to be declared, but I've changed the mechanism to read from the `module.lb` instead of the `board.xml`.

Reply:
Ok for me, but currently that does not happen in `module.lb`.